src/main/java/com/gradle/Overrides.java

```diff
@@ -99,4 +104,44 @@ private static String toEnvVarName(String sysPropertyName) {
         return sysPropertyName.toUpperCase().replace('.', '_');
     }
 
+    private static URI replacePath(URI uri, String path) {
+        try {
+            String finalPath = appendIfMissing(prependIfMissing("/", path), "/");
```

Comment:
I think this append(prepend("/", path), "/") pattern could be made a little clearer if refactored as one function call.

Reply:
So, a `prependAndAppendIfMissing`? +1

Reply:
That name sounds good to me.

Reply:
Done.